bundles/org.openhab.binding.velux/README.md

```diff
@@ -222,20 +227,20 @@ The bridge Thing provides the following properties.
 
 ```
 Bridge velux:klf200:g24 "Velux KLF200 Hub" @ "Under Stairs" [ipAddress="192.168.1.xxx", password="secret"] {
-    Thing window w56-36-13-5A-11-2A-05-70 "Bathroom Roof Window" @ "Bathroom" [serial="56:36:13:5A:11:2A:05:70", inverted=true]
+	// Velux (standard) window (with serial number)
+    Thing window Bathroom_Roof_Window "Bathroom Roof Window" @ "Bathroom" [serial="56:36:13:5A:11:2A:05:70", inverted=true]
+
+	// Somfy (non-standard) rollershutter (without serial number)
+    Thing rollershutter Living_Room_Awning "Living Room Awning" @ "Living Room" [serial="00:00:00:00:00:00:00:00", name="Living Room Awning"]
 }
 ```
 
-[=> download sample things file for textual configuration](./doc/conf/things/velux.things)
```

Comment:
You are suppressing links to files in folder doc/conf but you keep the files. 
Makes no real sense.

Reply:
> Why removing them?

Essentially the contents of the linked files is already included in the body of the read me file.

> but you keep the files.

Yeah. So far, I did not delete them "just in case". But I suppose that I should actually delete them too..

Reply:
On reflection, I have restored the links to the file. But I have deleted some old/bad ones.